src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs

```diff
@@ -333,8 +333,8 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                                     continue;
                                 }
 
-                                _logger.Error("Error while parsing field={0}, selector={1}, value={2}: {3}", field.Key, field.Value.Selector, value == null ? "<null>" : value, ex.Message);
-                                throw;
+                                //Parser errors usually happen on every result and are costly to performance, trace only
+                                _logger.Trace("Error while parsing field={0}, selector={1}, value={2}: {3}", field.Key, field.Value.Selector, value == null ? "<null>" : value, ex.Message);
```

Comment:
This make parsing extremely slow if any of the rules don't parse. Ex. Blutopia and posters